python/pyspark/sql/connect/expressions.py

```diff
@@ -162,7 +172,7 @@ def __init__(self, parent: Expression, alias: Sequence[str], metadata: Any):
 
     def to_plan(self, session: "SparkConnectClient") -> "proto.Expression":
         if len(self._alias) == 1:
-            exp = proto.Expression()
+            exp = proto.Expression(origin=self.origin)
```

Comment:
We should fix this repetitive code in a similar way we've done for relations by creating a common create method that avoids having to know how to initialize the common attributes of the class.